app/helpers/template_helpers.py

```diff
@@ -139,29 +146,39 @@ def _footer_warning(self) -> Optional[str]:
 
 
 @lru_cache
-def survey_config_mapping(theme: str, language: str) -> SurveyConfig:
-    return {
+def survey_config_mapping(*, theme: str, language: str, base_url: str) -> SurveyConfig:
+    survey_type_to_config: dict[str, Type[SurveyConfig]] = {
         "default": BusinessSurveyConfig,
         "business": BusinessSurveyConfig,
         "health": SurveyConfig,
         "social": SurveyConfig,
         "northernireland": SurveyConfig,
         "census": (WelshCensusSurveyConfig if language == "cy" else CensusSurveyConfig),
         "census-nisra": CensusNISRASurveyConfig,
-    }[theme]()
+    }
+
+    return survey_type_to_config[theme](
+        base_url=base_url,
```

Comment:
Is this `base_url` argument needed if `SurveyConfig` class takes `base_url` from env var?

Reply:
It needs to be passed in from the value we get in the claims, as the env var is a fallback default for when we don't have a cookie.

Reply:
Got it, thanks!

Reply:
This doesn't seem right , it is being flagged in pycharm as _unused argument_